t/core/config-default.t

```diff
@@ -91,3 +91,53 @@ apisix:
 GET /t
 --- response_body
 admin_key: null
+
+
+
+=== TEST 5: support listen multiple ports with array
```

Comment:
Need to add tests in t/grpc-proxy-test.sh.
And need to update the doc/grpc-proxy.md

Reply:
done